snaps_openstack/ansible_p/ansible_utils/proxy_data.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-no_proxy: ""
-https_proxy: ""
-http_proxy: ""
-ftp_proxy: ""
+no_proxy: "127.0.0.1,172.16.141.253,172.16.141.169,172.16.141.22,172.16.141.23,172.16.141.39,172.16.141.107"
```

Comment:
Please remove the proxy settings from the commit as we don't want to expose them in public repo.

Reply:
Done